python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -132,6 +136,8 @@ def post(self, mine_no_or_guid=None):
             self.raise_error(400, 'Error: {}'.format(e))
         mine_identity.save()
         mine_detail.save()
+        if mine_tenure_type_id:
+            self._save_mine_type(mine_identity, mine_tenure_type_id)
```

Comment:
Is this the same behaviour as MineType(model).create_mine_type? If so, can we remove this internal method?
If not, can we move the method to the model? Just so the resource doesn't have to worry about how the data actually is created?